docs/style-guide.adoc

```diff
@@ -415,6 +473,7 @@ RETURN map
 ----
 
 . No padding space for parameters.
+.. This rule mentions deprecated syntax. See link:https://github.com/opencypher/openCypher/blob/master/cip/2.testable/CIP2016-07-07-Parameter-syntax.adoc#interaction-with-existing-features[Parameter Syntax].
```

Comment:
In contrast to my comment about surrounding nodes with parentheses, this is one rule I would definitely keep for the time being 👍